src/main/java/com/vegaone/venus/domain/ProjectEntity.java

```diff
@@ -19,5 +19,7 @@ public class ProjectEntity {
     @JoinColumn(name = "company_id")
     @JsonIgnore
```

Comment:
Could you also please remove the @JsonIgnore adnotation? I don't think it's needed on the entity